docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -39,7 +39,9 @@ Additionally, this feature can be configured to require touch for every Teleport
     - Kubernetes Access
     - Desktop Access
 
-  To navigate these incompatiblities, we recommend enabling hardware key support only when necessary, such as for roles with access to critical infrastructure. These roles can be accessed as needed with Access Requests so that users can avoid these issues for their normal login sessions.
+  The unsupported features above will not function for users that have have hardware key support enforced. This is either because the user's hardware key can't be accessed within that feature (WebUI) or because the protocol only supports raw private keys currently (Agent forwarding, Kubernetes Access, Desktop Access). 
+
+  To navigate these incompatiblities, we recommend enabling hardware key support only when necessary, such as for roles with access to critical infrastructure. These roles can be accessed as needed with Access Requests so that users can avoid these issues for their normal login sessions. We also recommend trying Teleport Connect as an alternative to the WebUI.
```

Comment:
Just breaking up the lines:

```suggestion
  The unsupported features above will not function for users that have have hardware key support
  enforced. This is either because the user's hardware key can't be accessed within that feature
  (WebUI) or because the protocol only supports raw private keys currently (Agent forwarding,
  Kubernetes Access, Desktop Access). 

  To navigate these incompatiblities, we recommend enabling hardware key support only when
  necessary, such as for roles with access to critical infrastructure. These roles can be accessed
  as needed with Access Requests so that users can avoid these issues for their normal login
  sessions. We also recommend trying Teleport Connect as an alternative to the WebUI.
```